projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -2842,8 +2842,13 @@ function process_submission() {
 			$reply_to_addr = $comment_author_email;
 		}
 
-		$headers = 'From: "' . $comment_author . '" <' . $from_email_addr . ">\r\n" .
-		           'Reply-To: "' . $comment_author . '" <' . $reply_to_addr . ">\r\n";
+		$headers = sprintf(
+			'From: %1$s <%2$s>%4$sReply-To: %1$s <%3$s>%4$s',
+			$comment_author,
+			$from_email_addr,
+			$reply_to_addr,
+			"\r\n"
+		);
```

Comment:
Why are you passing the "\r\n" in as a parameter instead of including it in the string?
```suggestion
		$headers = sprintf(
			"From: %1\$s <%2\$s>\r\nReply-To: %1\$s <%3\$s>\r\n",
			$comment_author,
			$from_email_addr,
			$reply_to_addr
		);
```
Or if you don't like having to escape the `$`, go closer to the original code with fixed indenting:
```suggestion
		$headers = 'From: ' . $comment_author . ' <' . $from_email_addr . ">\r\n" .
			'Reply-To: ' . $comment_author . ' <' . $reply_to_addr . ">\r\n";
```

Reply:
I may have been trying to be too smart about it here. :) You're right. I'll go with solution 2, it's probably the more readable choice.